pennylane/tape/tape.py

```diff
@@ -274,8 +309,6 @@ def __init__(self, name=None, do_queue=True):
         self.all_sampled = False
```

Comment:
Cannot comment here, but in line 229 / 264 is a wrong indent that shows up in the docs...